maintenance-branch

```diff
@@ -1 +1 @@
-4-0-dev
\ No newline at end of file
+remove-monkey-patching
```

Comment:
Whats this dependant on?

Reply:
I
Core should be merged first.
Core's `4-0-dev` has this:
```ruby
        RSpec::Mocks.configuration.tap do |config|
          config.syntax = :expect
```
and would fail if this was merged first.
Core's `remove-monkey-patching`'s specs pass when run against `4-0-dev` of Mock and Expectations.

Mocks and Expectations' PRs can be merged in any order.

Reply:
Ah ok, this explanation resolves a point I had on core, in future it'd be useful if this sort of explicit information is in the PR description, or commented on the branch like this so people know why, I have an alternate solution.